dynamiq/connections/connections.py

```diff
@@ -201,10 +203,28 @@ def connect(self) -> "OpenAIClient":
         """
         # Import in runtime to save memory
         from openai import OpenAI as OpenAIClient
-        openai_client = OpenAIClient(api_key=self.api_key)
+
+        openai_client = OpenAIClient(api_key=self.api_key, base_url=self.url)
         logger.debug("Connected to OpenAI")
         return openai_client
 
+    @property
+    def conn_params(self) -> dict:
+        """
+        Returns the parameters required for connection.
+
+        Returns:
+            dict: A dictionary containing
+
+                -the API key with the key 'api_key'.
```

Comment:
We don't specify all keys in dicts in case of return. Here all clear without this info in docstring